module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -232,4 +232,56 @@ public function findContainingTheme($relativePath, $returnType = false)
 
         return null;
     }
+
+    /**
+     * Search the themes for a file pattern. Returns all matching files.
+     *
+     * Note that for any matching file the last match in the theme hierarchy is
+     * returned.
+     *
+     * @param string|array $relativePathPattern Relative path pattern (or array of
+     * patterns) to search within themes
+     *
+     * @return array
+     */
+    public function findInThemes($relativePathPattern)
```

Comment:
This new method seems like a good candidate for a unit test (and potentially for merging separately, prior to the rest of this code). If you'd like me to split it off and write the test, I'm willing to do that if you don't get there first.

Reply:
Test added. I'd be fine with merging it separately, though it feels a bit odd to add a method that's seemingly unused..

Reply:
When a new method is of general use and stands alone, I like to commit it separately just to keep the history cleaner (and to reduce cognitive load when reviewing commits). I'd rather see a small commit with a message like "Add new method to do X, to be used by feature Y" than to just have an "Add feature Y" commit on a shared utility class, which provides less context about why the file was touched when browsing history. But I'm just explaining myself, not asking for anything to be done differently; I'm happy to manually pull these things out as needed.

Reply:
@demiankatz Ok, makes sense!

Reply:
I've gone ahead and done this as commit 6a90d39. Thanks again!